uptane-standard.md

```diff
@@ -467,7 +455,7 @@ A repository's Root metadata distributes the public keys of the top-level Root,
 * A representation of the public keys for all four roles. Each key SHALL have a unique public key identifier.
 * An attribute mapping each role to (1) its public key(s), and (2) the threshold of signatures required for that role.
 
-Additionally, it MAY contain a mapping of roles to a list of valid URLs from which the role metadata can be downloaded.  If this mapping of URLs is used, the implementer SHOULD implement this functionality following {{TAP-5}} to avoid adding unforeseen security risks.
+Additionally, it MAY contain a mapping of roles to a list of valid URLs from which the role metadata can be downloaded.
```

Comment:
Hmm, then we must remove this option and sentence altogether, because it depends on TAP 5.

Reply:
Fair enough, done.